### PR TITLE
Update nGraph Python API build instructions

### PR DESCRIPTION
**Pull request by postrational**

### CODE REVIEW

File: ngraph/python/BUILDING.md [hunk 1]
```diff
@@ -1,76 +1,145 @@
...
+    cd openvino/build
+    
+    cmake .. \
+        -DENABLE_CLDNN=OFF \
+        -DENABLE_OPENCV=OFF \
+        -DENABLE_VPU=OFF \
+        -DENABLE_PYTHON=ON \
+        -DNGRAPH_PYTHON_BUILD_ENABLE=ON \
+        -DNGRAPH_ONNX_IMPORT_ENABLE=ON \
+        -DCMAKE_INSTALL_PREFIX="${MY_OPENVINO_BASEDIR}/openvino_dist"
```
Comment (mbencer) [inline ngraph/python/BUILDING.md:50]: Minor: I think that it is worth to add that user can specify used version of Python by `-DPYTHON_EXECUTABLE=`which python3.8`.

**Review by mbencer — APPROVED** [review]
LGTM. Great, that you added more details

File: ngraph/python/BUILDING.md [hunk 2]
```diff
@@ -1,76 +1,146 @@
...
 
-    `$ make python_wheel`
+    apt install git wget build-essential cmake
+    apt install python3 python3-dev python3-pip python3-virtualenv python-is-python3
+    pip3 install cython
 
-After this procedure completes, the `ngraph/build/python/dist` directory should
-contain the Python packages of the version you cloned. For example, if you
-checked out and built `0.21` for Python 3.7, you might see something like:
+You can see a full working example od an Ubuntu environment used in our continuous environment in this
```
Comment (rblaczkowski) [inline ngraph/python/BUILDING.md:18]:
```
You can see a full working example **od** an Ubuntu environment used in our continuous environment in this 
```
Typo?

File: ngraph/python/BUILDING.md [hunk 3]
```diff
@@ -1,76 +1,146 @@
...
 
-    $ ls python/dist/
-    ngraph-core-0.21.0rc0.tar.gz
-    ngraph_core-0.21.0rc0-cp37-cp37m-linux_x86_64.whl
+On MacOS you can use [Homebrew](https://brew.sh) to install required packages:
 
-## Building nGraph Python Wheels on Windows
+    brew install cmake
+    brew install automake
+    brew install libtool
```
Comment (rblaczkowski) [inline ngraph/python/BUILDING.md:25]: Proposition - to add also `pip3 install cython` to MacOS required packages list

File: ngraph/python/BUILDING.md (hunk 1 above)
Comment (rblaczkowski) [reply ngraph/python/BUILDING.md:50]: I just wanted to add the same comment as above - on MacOS (just testes) without specifying DPYTHON_EXECUTABLE, system uses python2 by default - and this is not supported version of python here

**Pull request merged**